### DISCUSSION TITLE
How to cast a database type to something else for the purpose of comparison?

### DISCUSSION THREAD

**Discussion by gajus** [ts-safeql/safeql#289] [General]
I am getting a lot of error like:

> Invalid Query: column "category" is of type notification_category but expression is of type text

I know I can fix it like,

```ts
${'OTHER'}::notification_category,
```

but there are thousands of these, and really not much value in adding this code.

Is there a way to configure that `notification_category` should be treated as text?

I think what I am asking is the inverse of https://safeql.dev/api/#connections-overrides-types-optional, and it appears to be not supported.

**Comment by Newbie012** [comment]
I think I ended up with a slightly better alternative that won't require any configuration https://github.com/ts-safeql/safeql/pull/294 It's still a draft, but at the current state it may solve your issue.

**Reply by Newbie012** [reply]
fixed in 3.5.0

**Reply by gajus** [reply]
Thank you ❤️
